src/frontend/hooks/update-record.ts

```diff
@@ -27,7 +31,7 @@ const updateRecord = (
   }
   // set new value
   if (value) {
-    if (typeof value === 'object') {
+    if (typeof value === 'object' && !(value instanceof File)) {
```

Comment:
this is the actual fix - before function wanted to unflatten the File object (when file was given)